catalyst_voices/packages/catalyst_voices_models/lib/src/seed_phrase.dart

```diff
@@ -88,6 +106,9 @@ class SeedPhrase {
     );
   }
 
-  /// The full list of BIP-39 mnemonic words in English.
-  static List<String> get wordList => WORDLIST;
+  @override
+  String toString() => 'SeedPhrase(****)';
```

Comment:
Can you please add hash instead of '****'? Would allow to compare two different seed phrases without exposing them